README.md

```diff
@@ -64,23 +64,24 @@ A curated list of awesome inclusive events/projects/initiatives for women in the
 ### UX
   - [Ladies that UX](http://ladiesthatux.com/)
 
-    Ladies that UX is a monthly meetup that creates a welcoming, transparent community of women that work in UX, who positively promote and teach each other.
+    Ladies that UX is a monthly meet-up that creates a welcoming, transparent community of women who work in UX, who positively promote and teach each other.
 
   - [XX-UX](http://www.xxux.org/)
 
+    XX+UX aims to build a supportive community that fosters diversity, support, empowerment, and mentorship for women.
     The first XX+UX event was held at Google HQ in October 2013, where attendees created “story cards” that represented themselves or their journey into UX.
 
 ### General
   - [Byte Girl](http://bytegirl.com.br/) - *Loc:* <img src="https://upload.wikimedia.org/wikipedia/en/0/05/Flag_of_Brazil.svg" alt="Brazil" width="30"> - *Lang: PT-BR only*
 
-    Byte Girl is a non-profit and open event organized by volunteers focused on the female public who likes tech. It has the goal of developing women empowerment.
+    Byte Girl is a non-profit, open event organized by volunteers focused on females who likes tech. It's goal is to empower women via talks from professionals who share valuable knowledge and experiences in technology.
```

Comment:
* The comma is not needed.

* `It's` should be `Its`

Reply:
I believe that a comma works between coordinate adjectives, but I fixed the its